li-protobuf/src/main/java/com/linkedin/data/protobuf/ProtoReader.java

```diff
@@ -49,47 +49,48 @@
 import java.io.EOFException;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
 
 /**
  * Utility class for reading Protocol Buffers encoded binary data.
  */
 public abstract class ProtoReader
```

Comment:
We copied because: 

- We want to modify them to work with our loose binding using symbol tables, as opposed to protobuf vanilla impl that depends on build time schemas and generated code.
- String parsing enhancements that improve string parsing performance beyond vanilla protobuf.
-

Reply:
Ok. How do you plan to get any future bug fix from Google? Hope that there are not many bugfix, otherwise it is a pain, that is how we deal with that URL jersey parser before in pegasus.